components/tooltip/demo/demo.html

```diff
@@ -0,0 +1,75 @@
+<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../coherent-gameface-components-theme.css">
+    <link rel="stylesheet" href="../style.css">
+    <title>Document</title>
+    <style>
+        .target {
+            width: 150px;
+            text-align: center;
+            background-color: #6e6d6d;
+            color: #ffffff;
+            position: relative;
+            top: 500px;
+            left: 500px;
+            padding: 10px;
+        }
+
+        .container {
+            display: flex;
+            flex-direction: column;
+        }
+
+        .title,
+        .button {
+            height: 40px;
+            background-color: var(--default-color-gray);
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+
+        .button {
+            background-color: var(--default-color-gray);
+            cursor: pointer;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="target">Hover over me</div>

```

Comment:
Second showing of the upper tooltip requires two clicks, reproduce:
Click on the target element to show the tooltip -> click **on the tooltip** to hide -> clicking once more on the target element is not showing the tooltip -> the second click is showing it.

Reply:
Yes, because hide does not update the visible state - fixed.